tensorflow/lite/micro/micro_profiler.cc

```diff
@@ -67,4 +67,58 @@ void MicroProfiler::LogCsv() const {
 #endif
 }
 
+void MicroProfiler::LogTicksPerTagCsv() {
+#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+  MicroPrintf(
+      "\"Unique Ops in the Graph\",\"Total Ticks (all instances of the Op)\"");
+  int totalTicks = 0;
+  for (int i = 0; i < num_events_; ++i) {
+    uint32_t ticks = end_ticks_[i] - start_ticks_[i];
+    int position = FindExistingOrNextPosition(tags_[i]);
+    total_ticks_per_tag[position].tag = tags_[i];
+    total_ticks_per_tag[position].ticks =
+        total_ticks_per_tag[position].ticks + ticks;
+    totalTicks += ticks;
+  }
+
+  for (int i = 0; i < num_events_; ++i) {
+    ticks_per_tag ticksPerTag = total_ticks_per_tag[i];
+    if (ticksPerTag.tag == nullptr) {
+      break;
+    }
+    MicroPrintf("%s,%d", ticksPerTag.tag, ticksPerTag.ticks);
+  }
+  MicroPrintf("total,%d", totalTicks);
```

Comment:
this seems to not strictly be a part of the csv format. I would say skip from the current function.

Reply:
Got it. I was thinking to print the total count as well as I have seen that this is required for the calculation but if you think that is not required, I can remove it. I can probably change it to "total number of ticks" or something like that to make it more meaningful.